src/controller/editingcontroller.js

```diff
@@ -61,6 +64,19 @@ export default class EditingController {
 		this.mapper = new Mapper();
 
 		/**
+		 * Defines whether controller is in read-only mode.
```

Comment:
I mean – don't describe the code. Describe what it means.

And I've just thought that perhaps this property isn't needed at all in the EditingController and should be added to the view Document only? Then we'll avoid this strange situation which we can't even properly document.